components/teleport/Teleport.vue

```diff
@@ -271,7 +271,6 @@ const getTransaction = async () => {
     return paraspell
       .Builder(api)
       .from(Chain[fromChain.value.toUpperCase()])
-      .currency('KSM')
```

Comment:
This is a breaking change introduced in the 2.0.5 version. It further simplifies call construction as transfers to Relay chain are either DOT or KSM. We do not need to enter currency because of that. We can directly find out Relay chain symbol for specific Parachain from our asset map.

Reply:
See this issue for reference https://github.com/kodadot/nft-gallery/issues/6431.
Para to Para scenario remains unchanged and has to have .currency (because of possibility to transfer other than DOT and KSM currencies across Parachains)

Reply:
Noted thanks!